### PR TITLE
jcabi-latex-maven-plugin execution to fail on Windows - Issue #9

### PR DESCRIPTION
**Pull request by Marczeeee**
Issue #9

### CODE REVIEW

**Comment by dmarkov** [comment]
Thanks, someone will review your pull request soon

**Comment by dmarkov** [comment]
@carlosmiranda could you please review this pull request

File: src/main/java/com/jcabi/latex/maven/plugin/CompileMojo.java [hunk 1]
```diff
@@ -141,4 +143,17 @@ private void compile(final Compiler compiler, final String name)
...
     }
 
+    /**
+     * Checks the type of the operating system running and failing if it's an
+     * unsupported one (like Windows) for this plugin.
+     * @throws MojoFailureException If an unsupported operation system is detected
+     */
+    private void checkOS() throws MojoFailureException {
+        final String osName = System.getProperty("os.name");
+        if (StringUtils.startsWith(osName, "Windows")) {
```
Comment (carlosmiranda) [inline src/main/java/com/jcabi/latex/maven/plugin/CompileMojo.java]: Let's use `SystemUtils.IS_OS_WINDOWS` from `commons-lang3` ([Javadoc](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/SystemUtils.html#IS_OS_WINDOWS))

File: src/main/java/com/jcabi/latex/maven/plugin/CompileMojo.java [hunk 2]
```diff
@@ -99,6 +100,7 @@ public void execute() throws MojoFailureException {
         if (this.skip) {
             Logger.info(this, "execution skipped because of 'skip' option");
         } else {
+            this.checkOS();
```
Comment (carlosmiranda) [inline src/main/java/com/jcabi/latex/maven/plugin/CompileMojo.java]: I don't think we need a separate method for this. It's a fairly simple check that we only do once, here.

**Comment by carlosmiranda** [comment]
@Marczeeee Thanks for the pull request, I have a couple of minor comments above.

**Comment by Marczeeee** [comment]
@carlosmiranda Updated the code according to your comments.

**Comment by carlosmiranda** [comment]
@Marczeeee Thank you very much for the update!

**Comment by carlosmiranda** [comment]
@rultor let's merge please

**Comment by carlosmiranda** [comment]
@yegor256 Could you help us out here please?

**Comment by yegor256** [comment]
@rultor merge it

**Comment by dmarkov** [comment]
@carlosmiranda done, I added **19 mins** in payment `50597587`. +19 added to your rating, current score is: [+1993](http://www.netbout.com/b/35023?open=rating)

**Comment by dmarkov** [comment]
@rultor please deploy

**Pull request merged**